README.md

```diff
@@ -1,2 +1,48 @@
-# consul-net-tools
-Experimental networking tools for consul; don't use
+# Consul net/rpc
+
+This repository contains a fork of the [net/rpc] package from the Go standard library. The codec
+used with `net/rpc` contains references to the `net/rpc` package, so this fork also contains
+a fork of [hashicorp/net-rpc-msgpackrpc] and [hashicorp/go-msgpack] so those references
+to `net/rpc` point at the correct package.
+
+**Important**: This fork was created exclusively for Consul. We will not accept
+any changes to this repository for any other use cases. We discourage any use of this
+library outside of Consul.
+
+[net/rpc]: https://pkg.go.dev/net/rpc
+[hashicorp/net-rpc-msgpackrpc]: https://github.com/hashicorp/net-rpc-msgpackrpc
+[hashicorp/go-msgpack]: https://github.com/hashicorp/go-msgpack
+
+## Why does this fork exist?
+
+[net/rpc] has been [frozen since late 2016], which means no changes are accepted to the
+package, and none of the existing bugs will be fixed. 
```

Comment:
```suggestion
[net/rpc] has been [frozen since late 2016], which means further contributions will be made to the package. Unless, they are security related. 
```

Reply:
hm, let's punt on this -- I need to find a ref/ link to the security support commitment, if any.